db/models.js

```diff
@@ -37,3 +33,25 @@ exports.fetchArticleById = (article_id) => {
       return article.rows[0];
     });
 };
+
+exports.fetchArticleComments = (article_id, sort_by = "created_at") => {
+  const sortableColumns = ["created_at"];
+
+  if (!sortableColumns.includes(sort_by)) {
+    return Promise.reject({ status: 400, message: "Invalid sort_by." });
+  }
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} DESC`,
```

Comment:
You don't need to pass in a sort_by for this task, the comments just need to be sorted by created_at descending so you can just create your query for that, it doesn't need to be dynamic.